### PR TITLE
Docs for Enterprise Hub

### PR DESCRIPTION
**Pull request by julien-c**
### TL;DR: adding an entry point re. all Enterprise Hub features

see the doc-build here: https://moon-ci-docs.huggingface.co/docs/hub/pr_1081/  (as soon as it will be built)

So we can point prospective customers to those docs

### a few Screenshots (but open the doc-build instead)

image

### CODE REVIEW

**Comment by mishig25** [comment]
looking into it. Somehow, the docs are rendering the main branch rather than the PR branch

**Comment by mishig25** [comment]
It is working https://moon-ci-docs.huggingface.co/docs/hub/pr_1081/en/billing no ?

image

**Comment by julien-c** [comment]
ah yes seems to work no 👍 (i had a 404)

**Comment by julien-c** [comment]
arg it failed now for some reason @mishig25

**Comment by mishig25** [comment]
You got 404 probably while the doc artists were being uploaded to moonci docs server

**Comment by Pierrci** [comment]
The Enterprise icon looks a bit weird/lonely to me, no? Did you mean to put it inline with the title?

image

Other than that, LGTM

**Comment by abidlabs** [comment]
Agree with @Pierrci, also the same issue is on the main Enterprise Hub page:

image

**Comment by abidlabs** [comment]
This screenshot is a bit too large:

image

Otherwise, screenshots are quite nice! LGTM

**Review by abidlabs — APPROVED** [review]

File: docs/hub/billing.md [hunk 1]
```diff
@@ -7,9 +7,12 @@ We also partner with cloud providers, like [AWS](https://huggingface.co/blog/aws
 All users and organizations accounts have a billing system set up. You can submit your payment info and then access "pay-as-you-go" services.
 
 From the [Settings > Billing](https://huggingface.co/settings/billing) page, you can see a real time view of your usage across different HF services:
```
Comment (jeffboudier) [inline docs/hub/billing.md:9]:
```suggestion
From the [Settings > Billing](https://huggingface.co/settings/billing) page, you can see a real time view of your paid usage across all HF services, for instance:
```

File: docs/hub/billing.md [hunk 2]
```diff
@@ -7,9 +7,12 @@ We also partner with cloud providers, like [AWS](https://huggingface.co/blog/aws
...
+
 - [Spaces](./spaces),
 - [AutoTrain](https://huggingface.co/docs/autotrain/index),
 - [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index)
+- [PRO subscriptions](https://huggingface.co/pricing) (for users)
```
Comment (jeffboudier) [inline docs/hub/billing.md:14]:
```suggestion
- [PRO subscription](https://huggingface.co/pricing) (for users)
```

File: docs/hub/billing.md [hunk 3]
```diff
@@ -7,9 +7,12 @@ We also partner with cloud providers, like [AWS](https://huggingface.co/blog/aws
...
+- [Enterprise Hub subscriptions](https://huggingface.co/enterprise) (for orgs)
```
Comment (jeffboudier) [inline docs/hub/billing.md:15]:
```suggestion
- [Enterprise Hub subscription](https://huggingface.co/enterprise) (for organizations)
```

File: docs/hub/enterprise-sso.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Single Sign-On (SSO)
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```
Comment (jeffboudier) [inline docs/hub/enterprise-sso.md:4]:
```suggestion
This feature is part of <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```

File: docs/hub/enterprise-sso.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+</Tip>
+
+Read the [documentation for SSO under our Security section](./security-sso).
```
Comment (jeffboudier) [inline docs/hub/enterprise-sso.md:7]:
```suggestion
Read the [documentation for SSO under the Security section](./security-sso).
```

File: docs/hub/storage-regions.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+# Storage Regions on the Hub
+
+Regions let you decide where your org's models and datasets will be stored.
```
Comment (jeffboudier) [inline docs/hub/storage-regions.md:3]:
```suggestion
Regions let you decide where your organization models and datasets will be stored.
```

File: docs/hub/storage-regions.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```
Comment (jeffboudier) [inline docs/hub/storage-regions.md:6]:
```suggestion
This feature is part of <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```

File: docs/hub/storage-regions.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+Currently we support the following regions:
+
+- US 🇺🇸
+- EU 🇪🇺
+- coming soon: Asia-Pacific 🌏
+
+## How to set up
+
+If your organization is an Enterprise Hub org, you will be able to see the Regions settings page:
```
Comment (jeffboudier) [inline docs/hub/storage-regions.md:22]:
```suggestion
If your organization is subscribed to Enterprise Hub, you will be able to see the Regions settings page:
```

File: docs/hub/storage-regions.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/feature-annotated.png)
+
+On that page you can see:
+
+- an audit of where your orgs' repos are currently located
```
Comment (jeffboudier) [inline docs/hub/storage-regions.md:28]:
```suggestion
- an audit of where your organization repos are currently located
```

File: docs/hub/storage-regions.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+## Repository Tag
+
+Any repo (model or dataset) stored in a non-default location will display its Region directly as a tag. That way your organization's members can see at a glance where repos are located.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/tag-on-repo.png)
+
+## Regulatory and legal compliance
+
+In many regulated industries, you may have a requirement to store your data in a specific area.
```
Comment (jeffboudier) [inline docs/hub/storage-regions.md:39]:
```suggestion
In regulated industries, companies may be required to store data in a specific region.
```

File: docs/hub/audit-logs.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+# Audit Logs
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" aria-hidden="true" focusable="false" role="img" width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 33 27"><path fill="currentColor" fill-rule="evenodd" d="M13.5.7a8.7 8.7 0 0 0-7.7 5.7L1 20.6c-1 3.1.9 5.7 4.1 5.7h15c3.3 0 6.8-2.6 7.8-5.7l4.6-14.2c1-3.1-.8-5.7-4-5.7h-15Zm1.1 5.7L9.8 20.3h9.8l1-3.1h-5.8l.8-2.5h4.8l1.1-3h-4.8l.8-2.3H23l1-3h-9.5Z" clip-rule="evenodd"></path></svg>
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```
Comment (jeffboudier) [inline docs/hub/audit-logs.md:6]:
```suggestion
This feature is part of <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
```

File: docs/hub/audit-logs.md [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+</Tip>
+
+Audit Logs enable organization admins to easily review actions taken by members, including organization membership, repository settings and billing changes.
+
+The resulting view is populated and accessible through your organization admin settings.
```
Comment (jeffboudier) [inline docs/hub/audit-logs.md:11]:
```suggestion
Audit Logs are accessible through your organization admin settings.
```

**Review by jeffboudier — APPROVED** [review]
Thx for adding this!

**Review by LysandreJik — APPROVED** [review]
Looks great! Thanks!

**Comment by julien-c** [comment]
@Pierrci @abidlabs yep i wanted to inline the icon but there's no easy way in markdown to inline an image, correct?

i'll need to write a bit of HTML i guess

**Comment by julien-c** [comment]
i'll merge this today or tomorrow, time for final reviews!

**Comment by julien-c** [comment]
@abidlabs ceb0d9eedc85821ca9857684a024f41a7aed911d

**Comment by julien-c** [comment]
ok looks quite good now with a small inline icon for Enterprise, so let's merge
image

**Pull request merged**